Makefile

```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
 # version is moved to a separate repo and release process.
-export IMAGE_VERSION = v1.4.0
+export IMAGE_VERSION = v1.5.0
 # Build-time variables to inject into binaries
```

Comment:
what about doing the release after Monday meeting and give a chance for the community to say if it has something that should be added before it rollout?

Reply:
This release has already been delayed for awhile. All issues in the v1.5.0 milestone have been completed, so there's no reason to block. Feel free to bring up your concerns in the next grooming meeting.